mkdocs.yml

```diff
@@ -51,6 +51,8 @@ nav:
             - Animations: guides/hedgehog-engine/blueblur/files/xncp/anims.md
         - Level Editing:
           - Sonic GLvl: guides/hedgehog-engine/blueblur/levels/glvl.md
+          - Importing Terrain: guides/hedgehog-engine/blueblur/levels/importing-terrain/importing-terrain.md
+          - Importing Collision: guides/hedgehog-engine/blueblur/levels/importing-collision/importing-collision.md
```

Comment:
You don't need a dedicated folder for these guides. You can place them directly in the `levels` folder, and place their images in the already existing `assets` folder there. The assets however do need their own dedicated folder.

TLDR: Move the guide up one level in the directory tree, and delete their dedicated folders